http/src/main/resources/init-resources/cluster-site-gce.conf

```diff
@@ -34,6 +34,7 @@
     # Append SameSite=None to cookies set by RStudio. This is required by some browsers because we
     # render RStudio in an iframe. There does not appear to be a way within RStudio to do this, hence
     # doing it in the proxy.
+    # Header edit Set-Cookie ^(.*)$ $1;Secure;SameSite=None;HttpOnly;Partitioned "expr=%{REQUEST_URI} =~ m#/proxy/[^/]*/[^/]*/rstudio/.*#"
```

Comment:
is this needed? Its commented out

Reply:
I left the places we'd need to change to get full CHIPS support as comments so that it's easy to implement if this does work.

Reply:
maybe just add a comment to indicate what this is in the relevant places

Reply:
This isn't meant to stick around, I just can't test it anywhere but dev! If it works, I remove the comment and ship this line; if it doesn't, I revert the commit.